monitoring/router.nginx.conf

```diff
@@ -1,20 +1,45 @@
 server {
-    server_name _;
+    server_name grafana.*;
 
     location /monitoring/grafana/ {
         proxy_pass http://grafana/;
-        proxy_set_header Host internal.hail.is;
+        proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Proto https;
```

Comment:
This should have been set upstream by gateway based on the original request.  You probably don't want to overwrite them.  $host is now probably grafana.internal, which is only used for internal routing.